tests/desktop/consumer_pages/test_users_account.py

```diff
@@ -100,11 +109,6 @@ def test_editing_user_profile(self, mozwebqa):
         profile_page.refresh_page()
         Assert.equal(profile_page.display_name, name)
 
-        # Undo the changes
-        profile_page.edit_display_name(initial_value)
-        profile_page.save_changes()
-        profile_page.wait_notification_box_not_visible()
-
     @pytest.mark.credentials
     def test_that_checks_changing_language(self, mozwebqa):
```

Comment:
I know it isn't related to this PR, but is there a reason we are keeping this test around? Will changing languages be implemented in fireplace at some point in the future? @krupa what do you think?